src/Compilers/CSharp/Portable/Generated/BoundNodes.xml.Generated.cs

```diff
@@ -10515,11 +10521,7 @@ internal abstract partial class BoundTreeWalker : BoundTreeVisitor
             this.VisitList(node.NamedArguments);
             return null;
         }
-        public override BoundNode? VisitUnconvertedObjectCreationExpression(BoundUnconvertedObjectCreationExpression node)
-        {
-            this.VisitList(node.Arguments);
```

Comment:
moving to draft.  looks liek teh compiler has special knowledg eof walking into ImmutableArrays of bound nodes as children.